### DISCUSSION TITLE
Is there a mechanism to delay a reclaimed Child ID be allocate to a new Child device?

### DISCUSSION THREAD

**Discussion by jinpeng1989** [openthread/openthread#7775] [Q&A] [answered]

**Comment by jwhui** [comment, marked as answer]
Thread does not provide a mechanism to control how reclaimed Child IDs are allocated.

OpenThread will attempt to cycle through the set of available Child IDs before reusing previously allocated Child IDs.
https://github.com/openthread/openthread/blob/40484c94687163e60e44e3f23855b49e7dd911df/src/core/thread/mle_router.cpp#L3156-L3168
